aikau/src/main/resources/alfresco/editors/TinyMCE.js

```diff
@@ -209,6 +243,64 @@ define(["dojo/_base/declare",
          } else {
             this._delayedInitConfig = config;
          }
+
+         if (this.autoResize)
+         {
+            this.alfSetupResizeSubscriptions(this.onResize, this);
+         }
+      },
+
+      /**
+       * When [autoResize]{@link module:alfresco/editors/TinyMCE} is configured to true this will
+       * respond to resize events by finding the first ancestor with height and width dimensions
+       * and then increasing the size of the TinyMCE editor to fill the available space as best
+       * it can.
+       * 
+       * @instance
+       * @since 1.0.47
+       */
+      onResize: function alfresco_editors_TinyMCE__onResize() {
+         if (this.autoResize)
+         {
+            if (this._editorInitialized)
+            {
+               // Work our way back up through the current node ancestors to find
+               // the first ancestor that have height and width set on them
+               var height, width;
```

Comment:
IndexOf on a style seems a bit strange. Also the code seems a bit off ... it looks like you'll end up with the first height you find and the first width you find, even if not on the same node, which might be intended but isn't what the comment says.

I'd have done this ...

```javascript
var parent = this.domNode,
   size;
while ((parent = parent.parentNode) && !size) {
   if (parent.style.height && parent.style.width) {
      size = {
         h: parent.style.height,
         w: parent.style.width
      };
   }
}
```

Up to you if you want to use any of it, or if you're happy with it as-is, but I'm not sure it's right at the moment.